audit/projects/k8s-infra-prow-build-trusted/secrets/cncf-ci-github-token/description.json

```diff
@@ -0,0 +1,10 @@
+{
+  "createTime": "2021-02-11T04:21:30.200768Z",
+  "labels": {
+    "sig": "testing"
+  },
```

Comment:
note to self for followup: need to decide on a convention for labeling secrets

I've been labeling other secrets in kubernetes-public with e.g. `labels.app=slack-infra`

I had considered `group=sig-testing` but feel like `sig=testing` lines up with "only sigs can own code"